src/javascript/main.js

```diff
@@ -0,0 +1,60 @@
+var Minispot = window.Minispot || {};
+
+var response;
+
+Minispot.search = function() {
+
+	var search_button = document.querySelector('#search');
+
+	search_button.addEventListener('click', function() {
```

Comment:
if instead of listening for the `click` event on the submit button you listen to the `submit` event in the form, you can also "hijack" when the user presses enter.